src/guides/v2.3/config-guide/multi-site/ms_apache.md

```diff
@@ -1,6 +1,6 @@
 ---
 group: configuration-guide
-title: Tutorial&mdash;Set up multiple websites with Apache
+title: Tutorial—Set up multiple websites with Apache
```

Comment:
Revert this. Long term, it would be better if the title were changed so that that there's no em dash.

Reply:
**done**